From b7354f5bc5fac1aedb369b069e661d233111e419 Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Thu, 06 Feb 2025 14:21:12 +0200
Subject: [PATCH] [SW-217978] g3: ignore td2,3 in hl_sensors.conf

HL325 from PCB R0C, should not display temp TD2 and TD3 sensors.
Since all PCB are R0C or newer, there is no need to handle this issue
specifically for PCB R0C.

Change-Id: I3c2c7292038df33cdb6014f36c57a60c23c47c01
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/package/sensors/hl_sensors.conf b/package/sensors/hl_sensors.conf
index 59bf4fd..cddf3cd 100644
--- a/package/sensors/hl_sensors.conf
+++ b/package/sensors/hl_sensors.conf
@@ -271,8 +271,8 @@
 label temp64 "TMP PCB D1"
 label temp65 "On Chip TD 0"
 label temp66 "On Chip TD 1"
-label temp67 "On Chip TD 2"
-label temp68 "On Chip TD 3"
+ignore temp67
+ignore temp68
 label temp69 "On Chip TD 4"
 label temp70 "On Chip TD 5"
 label temp71 "On Chip TD 6"
@@ -388,8 +388,8 @@
 label temp64 "TMP PCB D1"
 label temp65 "On Chip TD 0"
 label temp66 "On Chip TD 1"
-label temp67 "On Chip TD 2"
-label temp68 "On Chip TD 3"
+ignore temp67
+ignore temp68
 label temp69 "On Chip TD 4"
 label temp70 "On Chip TD 5"
 label temp71 "On Chip TD 6"
